### ISSUE TITLE
stochastic prediction dropout mask greater than one

### ISSUE THREAD

**Issue by daxiaoluo** [namisan/mt-dnn#22]
Thanks for your work and we have a problem with the stochastic prediction dropout.
```pythoh
def generate_mask(new_data, dropout_p=0.0, is_training=False):
    if not is_training: dropout_p = 0.0
    new_data = (1-dropout_p) * (new_data.zero_() + 1)
    for i in range(new_data.size(0)):
        one = random.randint(0, new_data.size(1)-1)
        new_data[i][one] = 1
    print(new_data)
    mask = Variable(1.0/(1 - dropout_p) * torch.bernoulli(new_data), requires_grad=False)
    return mask
```
When call this function as above, the element in the matrix will be greater than one, I want to known why multiply 'Variable(1.0/(1 - dropout_p)'. It will cause the element greater than one.
